precompile/contracts/deployerallowlist/config.go

```diff
@@ -57,3 +58,33 @@ func (c *Config) Equal(cfg precompileconfig.Config) bool {
 func (c *Config) Verify(chainConfig precompileconfig.ChainConfig) error {
 	return c.AllowListConfig.Verify(chainConfig, c.Upgrade)
 }
+
+func (c *Config) ToBytes() ([]byte, error) {
+	p := wrappers.Packer{
+		Bytes:   []byte{},
+		MaxSize: 32 * 1024,
+	}
+
+	if err := c.AllowListConfig.ToBytesWithPacker(&p); err != nil {
+		return nil, err
+	}
+
+	if err := c.Upgrade.ToBytesWithPacker(&p); err != nil {
+		return nil, err
+	}
+
+	return p.Bytes, nil
+}
+
+func (c *Config) FromBytes(bytes []byte) error {
+	p := wrappers.Packer{
+		Bytes: bytes,
+	}
+	if err := c.AllowListConfig.FromBytesWithPacker(&p); err != nil {
```

Comment:
Allowlist is not a config, instead it is a sub-config. It can be done that way I just thought it was more efficient to give a reference to the parker, otherwise we would have to know exactly how big the slice of bytes to share.

Reply:
Done in 4591ccdb